tfhe/src/core_crypto/entities/seeded_ggsw_ciphertext.rs

```diff
@@ -273,6 +273,27 @@ impl<Scalar: UnsignedInteger, C: Container<Element = Scalar>> SeededGgswCipherte
         decompressed_ct
     }
 
+    /// Parllel variant of
+    /// [`decompress_into_ggsw_ciphertext`](`Self::decompress_into_ggsw_ciphertext`)
+    pub fn par_decompress_into_ggsw_ciphertext(self) -> GgswCiphertextOwned<Scalar>
```

Comment:
Take by `&self`  as using `self` this can force for the caller the need to clone, when in reality,
the function only needs `&self`